src/db/mysql/models.rs

```diff
@@ -524,12 +532,12 @@ impl MysqlDb {
             Sorting::Oldest => query.order(bso::id.asc()).order(bso::modified.asc()),
             */
             Sorting::Index => query.order(bso::sortindex.desc()),
```

Comment:
maybe we should revert to the commented out issue559 code above? (does the same but also secondary sorts for Sorting::Index + None)

Reply:
Yeah, I was wondering about that and was going to ask you. Just forgot will all the other crap. Thanks!

Reply:
I think I may roll this change into the patch for #874, since it's impacting a test there.